File: 2º/PDOO/Practias/Napakalaki/src/napakalaki/Prize.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package napakalaki;

/**
 *
 * @author arturo
 */
public class Prize {
    private int treasures, level;
    Prize(int treasures, int level){
        this.treasures=treasures;
        this.level=level;                   
    }
    public int getTreasures(){
        return treasures;
    }
    public int getLevel(){
        return level;
    } 

    @Override
    public String toString() {
        return "Prize{" + "treasures=" + treasures + ", level=" + level + '}';
    }
    
}

File: 2º/PDOO/Practias/Napakalaki-ruby/lib/prize.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

class Prize
 
  attr_reader :treasures #Consultor forma 1
  def level #Consultor forma 2
    @level
  end
  
  
  def initialize(treasures,level) 
      @treasures = treasures
      @level = level
  end  
  
  def to_s
    "Tesoros ganados: #{@treasures} Niveles ganados: #{@level}"
  end
  
end
